### PR TITLE
Update fixtures documentation for publishing pages

### PR DESCRIPTION
**Pull request by matt-halliday**
The current documentation still requires the dev to log into the admin and publish pages despite setting the workflow to published. This addition publishes the pages.

| Q | A
| --- | ---
| Fixed tickets | fixes #issuenum
| License | MIT

#### What's in this PR?

Just a small update to actually publish pages added/updated by fixtures.

#### Why?

Had to dig into the code to figure out why the pages weren't actually being published when working from the documentation. Found that there was a step missing.

### CODE REVIEW

File: reference/components/document-manager/fixtures.rst [hunk 1]
```diff
@@ -55,6 +55,7 @@ Shown below is the simple data fixtures:
             $documentManager->persist($document, 'en', array(
                 'parent_path' => '/cmf/sulu_io/contents',
             ));
+            $documentManager->publish($document, 'en');
```
Comment (wachterjohannes) [inline reference/components/document-manager/fixtures.rst]: add a comment that this line is optional

File: reference/components/document-manager/fixtures.rst [hunk 2]
```diff
@@ -55,6 +55,10 @@ Shown below is the simple data fixtures:
             $documentManager->persist($document, 'en', array(
                 'parent_path' => '/cmf/sulu_io/contents',
             ));
+            
+            # Optional: If you don't want your document to be published, remove this line
```
Comment (matt-halliday) [inline reference/components/document-manager/fixtures.rst]: Not sure if you want this worded differently?

Comment (wachterjohannes) [inline reference/components/document-manager/fixtures.rst]: no that sounds good - thanks

**Comment by oafx** [comment]
huh, i thought i did fix that with #307 ... @wachterjohannes  OT question: what is the difference between your master and your develop branches and for what branch should i create future pull request for?

**Comment by wachterjohannes** [comment]
the develop branch contains new features which are developed in dev-develop of sulu. master represents the latest release

**Pull request merged**